webpack/services/api/index.js

```diff
@@ -103,3 +103,5 @@ export const foremanTasksApi = new ForemanTasksApi();
 const orgNode = document.getElementById('organization-id');
 // This node does not exist while testing
 export const orgId = orgNode ? orgNode.dataset.id : '1';
+const userNode = document.getElementById('user-id');
+export const userId = userNode ? userNode.dataset.id : '1';
```

Comment:
I know you just copied it from the line above, but is this code safe? Shouldn't we rather mock this in tests and raise error when `orgNode` or `userNode` are missing to avoid unexpected behavior in production?
@waldenraines, @sharvit what do you think?